src/test/java/org/jabref/logic/integrity/PredatoryJournalCheckerTest.java

```diff
@@ -0,0 +1,77 @@
+package org.jabref.logic.integrity;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.jabref.logic.journals.PredatoryJournalLoader;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class PredatoryJournalCheckerTest {
+
+    static PredatoryJournalChecker checker;
+    BibEntry entry;
+
+    @BeforeAll
+    static void initChecker() {
+        checker = new PredatoryJournalChecker(PredatoryJournalLoader.loadRepository(),
+                List.of(StandardField.JOURNAL, StandardField.PUBLISHER, StandardField.BOOKTITLE));
+    }
+
+    @BeforeEach
+    void initEntry() {
+        entry = new BibEntry();
+    }
+
+    @Test
+    void journalIsNotPredatory() {
+        entry.setField(StandardField.JOURNAL, "IEEE Software");
```

Comment:
I would propose to rewrite the entry code to follows

1. remove global variable `entry`
2. Change to `BibEntry entry = new BibEntry().withField(...)`

This makes the test code more local.